common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -257,4 +257,31 @@ public Properties cryptoConf() {
     return CryptoUtils.toCryptoConf("spark.network.crypto.config.", conf.getAll());
   }
 
+  /**
+   * When memory usage of Netty is above this water mark, it's regarded as memory shortage.
+   */
+  public Long nettyMemWaterMark() {
+    return JavaUtils.byteStringAsBytes(conf.get("spark.network.netty.memCostWaterMark", "1g"));
```

Comment:
oh, `core` module relies on `spark-network-common`, so we don't have to change here.